ballerina/Ballerina.toml

```diff
@@ -26,3 +26,16 @@ groupId = "javax.jms"
 artifactId = "javax.jms-api"
 version = "2.0.1"
 path = "./lib/javax.jms-api-2.0.1.jar"
+
+
```

Comment:
This space is not necessary?

Reply:
Fixed with [ed152d70a0ef75d15081cb276f282d6e99103d55](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/82/commits/ed152d70a0ef75d15081cb276f282d6e99103d55)